src/settings_gui.cpp

```diff
@@ -175,6 +175,32 @@ static StringID VolumeMarkFunc(int, int mark, int value)
 	return STR_GAME_OPTIONS_VOLUME_MARK;
 }
 
+static const int _min_small_font_size = 6;
+static const int _max_small_font_size = 20;
+
+static const int _min_medium_font_size = 8;
+static const int _max_medium_font_size = 22;
+
+static const int _min_large_font_size = 18;
+static const int _max_large_font_size = 32;
+
+static const int _min_monospaced_font_size = 8;
+static const int _max_monospaced_font_size = 22;
```

Comment:
Using something like

```
FontCache::GetDefaultFontHeight(fs) * smallest_factor
FontCache::GetDefaultFontHeight(fs) * largest_factor
```

would be better than these arbitrary hard coded values.

Reply:
Replace one set of hard coded numbers with a different set of hard coded numbers?

In effect it would be a trade off  between tying these numbers to the list in fontcache against the extra code for that. Is that a trade off you think is worth making?

Would each font have it's own factor scales here?  Medium and mono sort of share sizing.

Reply:
The constants are no more.